mqemitter-redis.js

```diff
@@ -18,8 +18,13 @@ function MQEmitterRedis (opts) {
 
   this._opts = opts
 
-  this.subConn = new Redis(opts.connectionString || opts)
-  this.pubConn = new Redis(opts.connectionString || opts)
+  if (opts.cluster && Array.isArray(opts.cluster)) {
```

Comment:
I don't see how this branch is triggered

Reply:
Would you like to see an integration test with a redis cluster in the workflow?   If so, would you be open to reducing scope to only allow for external connection?

Reply:
Yes